include/triton/core/tritonserver.h

```diff
@@ -592,7 +592,10 @@ typedef enum tritonserver_traceactivity_enum {
   TRITONSERVER_TRACE_COMPUTE_INPUT_END = 3,
   TRITONSERVER_TRACE_COMPUTE_OUTPUT_START = 4,
   TRITONSERVER_TRACE_COMPUTE_END = 5,
-  TRITONSERVER_TRACE_REQUEST_END = 6
+  TRITONSERVER_TRACE_REQUEST_END = 6,
+  TRITONSERVER_TRACE_TENSOR_QUEUE_INPUT = 7,
+  TRITONSERVER_TRACE_TENSOR_BACKEND_INPUT = 8,
+  TRITONSERVER_TRACE_TENSOR_BACKEND_OUTPUT = 9
 } TRITONSERVER_InferenceTraceActivity;
 
 /// Get the string representation of a trace activity. The returned
```

Comment:
Above need to increase TRITONSERVER_API_VERSION_MINOR by 1 because you have changed the API.

Reply:
TRITONSERVER_API_VERSION_MINOR is set from 6 to 7